worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -153,226 +153,273 @@ public BrushCommands(WorldEdit worldEdit) {
+        aliases = {
+            "cat",
+            "gravityline",
+            "saggedline"
+        },
+        desc = "Create a hanging line between two points"
+    )
     @CommandPermissions("worldedit.brush.spline")
     public void catenaryBrush(InjectedValueAccess context,
-        @Arg(desc = "Pattern") Pattern fill, @Arg(def = "1.2",
-                                                  desc = "Length of wire compared to distance between points")
-    @Range(from = 1,
-           to = Integer.MAX_VALUE) double lengthFactor,
-        @Arg(desc = "The radius to sample for blending",
-             def = "0") Expression radius, @Switch(name = 'h',
-                                                   desc = "Create only a shell") boolean shell,
-        @Switch(name = 's',
-                desc = "Select the clicked point after drawing") boolean select, @Switch(name = 'd',
-                                                                                         desc = "sags the catenary toward the facing direction")
-        boolean facingDirection) throws WorldEditException {
+                              @Arg(desc = "Pattern")
+                                  Pattern fill,
+                              @Arg(def = "1.2", desc = "Length of wire compared to distance between points")
+                              @Range(from = 1, to = Integer.MAX_VALUE)
+                                  double lengthFactor,
+                              @Arg(desc = "The radius to sample for blending", def = "0")
+                                  Expression radius,
+                              @Switch(name = 'h', desc = "Create only a shell")
+                                  boolean shell,
+                              @Switch(name = 's', desc = "Select the clicked point after drawing")
+                                  boolean select,
+                              @Switch(name = 'd', desc = "sags the catenary toward the facing direction")
+                                  boolean facingDirection) throws WorldEditException {
         worldEdit.checkMaxBrushRadius(radius);
         Brush brush = new CatenaryBrush(shell, select, facingDirection, lengthFactor);
         set(context, new CatenaryBrush(shell, select, facingDirection, lengthFactor))
             .setSize(radius).setFill(fill);
     }
 
-    @Command(name = "surfacespline",
-             aliases = {
-                 "sspline",
-                 "sspl"
-             },
-             desc = "Draws a spline (curved line) on the surface",
-             descFooter = "Create a spline on the surface\n"
-                 + "Video: https://www.youtube.com/watch?v=zSN-2jJxXlM")
+    @Command(
+        name = "surfacespline",
+        aliases = {
+            "sspline",
+            "sspl"
+        },
+        desc = "Draws a spline (curved line) on the surface",
+        descFooter = "Create a spline on the surface\n"
+            + "Video: https://www.youtube.com/watch?v=zSN-2jJxXlM"
```

Comment:
```suggestion
            + "Video: https://www.youtube.com/watch?v=CukvVOi4jv8"
```
The video used is dead.